karma.conf.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (config) {
         files: [
             { pattern: 'node_modules/babel-polyfill/browser.js' },
             { pattern: "src/**/*.ts" },
-            { pattern: "spec/**/*.spec.ts" }
+            { pattern: "test/**/*.spec.ts" }
```

Comment:
Why the change from `spec` to `test`? Is this a convention you prefer, or is there a behavior difference too?

Reply:
Works the same, it is used by aurelia repos.
My own thoughts about it, the folder test contains them and *.spec.ts is a way to describe them.

Reply:
Sure. I am going to push to master the changes that you suggested. This might lead to some conflicts. Let me know if that is a problem. For instance, I will use `test` instead of the `spec` directory, and will change double quotes to single quotes.